From 3bc0c6450ced1974375a52cb3849fdd589cb2563 Mon Sep 17 00:00:00 2001
From: ouni666 <fangwenbin@vip.qq.com>
Date: Thu, 25 Sep 2025 11:55:51 +0800
Subject: [PATCH] fix use system premake build

---
 build/workspaces/update-workspaces.sh | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/build/workspaces/update-workspaces.sh b/build/workspaces/update-workspaces.sh
index 05c7a2eb09..78334a0d01 100755
--- a/build/workspaces/update-workspaces.sh
+++ b/build/workspaces/update-workspaces.sh
@@ -46,10 +46,10 @@ echo
 echo "Premake args: ${premake_args}"
 if [ "$OS" != "Darwin" ]; then
 	# shellcheck disable=SC2086
-	${premake_command} --file="premake5.lua" --outpath="../workspaces/gcc/" ${premake_args} gmake2 || die "Premake failed"
+	${premake_command} --file="premake5.lua" --outpath="../workspaces/gcc/" ${premake_args} gmake || die "Premake failed"
 else
 	# shellcheck disable=SC2086
-	${premake_command} --file="premake5.lua" --outpath="../workspaces/gcc/" --macosx-version-min="${MIN_OSX_VERSION}" ${premake_args} gmake2 || die "Premake failed"
+	${premake_command} --file="premake5.lua" --outpath="../workspaces/gcc/" --macosx-version-min="${MIN_OSX_VERSION}" ${premake_args} gmake || die "Premake failed"
 	# Also generate xcode workspaces if on OS X
 	# shellcheck disable=SC2086
 	${premake_command} --file="premake5.lua" --outpath="../workspaces/xcode4" --macosx-version-min="${MIN_OSX_VERSION}" ${premake_args} xcode4 || die "Premake failed"
--
2.51.0
